rpm/389-ds-base.spec.in

```diff
@@ -572,6 +615,18 @@ cp -pa README ../%{name}-%{version}%{?prerel}/README.jemalloc
 popd
 %endif
 
+%if %{bundle_libdb}
+pushd ../%{libdb_base_version}
+libdbbuilddir=$PWD/BUILD/%{libdb_base_version}
+libdbdestdir=$PWD/../%{name}-%{version}%{?prerel}
+cp -pa $libdbbuilddir/LICENSE $libdbdestdir/LICENSE.libdb
+cp -pa $libdbbuilddir/README $libdbdestdir/README.libdb
+cp -pa $libdbbuilddir/lgpl-2.1.txt $libdbdestdir/lgpl-2.1.txt.libdb
+cp -pa $libdbbuilddir/dist/dist-tls/.libs/%{libdb_bundle_name} $RPM_BUILD_ROOT%{_libdir}/%{libdb_bundle_name}
```

Comment:
Can we put it in `%{_libdir}/%{pkgname}/lib` instead? I see that SONAME is different, but for consistency it would be good to keep all bundled libraries in one place.

Reply:
Not sure we can do that: if I simply change the path, ns-slapd is not able to load the bdb plugin because it does not found the lib (and no idea how to specify an rpath. BTW --disable-rpath in spec file let me think that rpath are not even used)
jemalloc is a very special case because it is not linked with the product but explicitly preloaded with LD_PRELOAD 
   (that provides the full path to the library)

That said I will put the library with the other ds shared libraries (like libslapd.so) in %{_libdir}/%{pkgname} 
(aka /usr/lib64/dirsrv/libdb-5.3-389ds.so)
I do not know what is the magic that allow to find that path :wink: , but that works !

Reply:
Ah yes, you're right about jemalloc, I forgot to that we LD_PRELOAD it.
And `/usr/lib64/dirsrv/` sounds good to me, it gives a clear message that it's not a system library, but one used by dirsrv.